source/_components/emulated_hue.markdown

```diff
@@ -49,37 +49,54 @@ emulated_hue:
-
-- **exposed_domains** (*Optional*): The domains that are exposed by default if `expose_by_default` is set to true. If not specified, this defaults to the following list:
-  - `switch`
-  - `light`
-  - `group`
-  - `input_boolean`
-  - `media_player`
-  - `fan`
-
-- **entities** (*Optional*): Customization for entities.
+{% configuration %}
+type:
+  description: The type of assistant which we are emulating. Either `alexa` or `google_home`. **This configuration option is deprecated and will be removed in a future release. It is no longer necessary to define type.**
+  required: false
+  type: string
+  default: google_home
+host_ip:
+  description: The IP address that your Home Assistant installation is running on. If you do not specify this option, the component will attempt to determine the IP address on its own.
+  required: false
+  type: string
+listen_port:
+  description: The port the Hue bridge API web server will run on. This can be any free port on your system.
+  required: false
+  type: integer
+  default: 8300
+advertise_ip:
+  description: If you need to override the IP address used for UPnP discovery. (For example, using network isolation in Docker)
+  required: false
+  type: string
+advertise_port:
+  description: If you need to specifically override the advertised UPnP port.
+  required: false
+  type: integer
+upnp_bind_multicast:
+  description: Whether or not to bind the UPnP (SSDP) listener to the multicast address (239.255.255.250) or instead to the (unicast) host_ip address specified above (or automatically determined). In special circumstances, like running in a FreeBSD or FreeNAS jail, you may need to disable this.
+  required: false
+  type: boolean
+  default: true
+off_maps_to_on_domains:
+  description: The domains that maps an "off" command to an "on" command. For example, if `script` is included in the list, and you ask Alexa to "turn off the *water plants* script," the command will be handled as if you asked her to turn on the script.
+  required: false
+  type: list
+  default: [script, scene]
+expose_by_default:
+  description: "Whether or not entities should be exposed via the bridge by default instead of explicitly (see the ‘emulated_hue’ customization below). Warning: If you have a lot of devices (more than 49 total across all exposed domains), you should be careful with this option. Exposing more devices than Alexa supports can result in it not seeing any of them. If you are having trouble getting any devices to show up, try disabling this, and explicitly exposing just a few devices at a time to see if that fixes it."
```

Comment:
Had to quote this due to special character in text ("Warning:")

Reply:
My apologies, @fredrikbaberg . I thought the quotes were going to be shown at the resulting page. Sorry you had to do unnecessary work.

Reply:
I understood the documentation as both being possible, but the quotes are required when there are control characters (https://developers.home-assistant.io/docs/en/documentation_standards.html). The example I started to look at used quotes for everything (and of course I can't find that now, so either I misread or it's been updated).
After your comment regarding quotes I'm only using them where required, and this is my interpretation of "add a note" (see above link to documentation standards).

Reply:
IMO the paragraph you provided only covers the immediate user's YAML config files, not the descriptions for internal use (like this). "Add a note" implies informing a user that if he does not quote particular values in his config, it may be invalid.

Reply:
Learning lot of new things this week, then I'll use quotes when needed in documentation.
You think the PR is OK as it is? Anything I should correct?